2024/opmodes/TwoStickTeleop.java

```diff
@@ -120,7 +124,12 @@ public void init() {
         junctionAligner.setPosition(.2);
         armServo.setPosition(ARM_FRONT);
 
-        liftMotor.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
+        // the motor must be at its set position zero, at the beginning of the opmode
+        hangingMotor.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
```

Comment:
Instead of setting the `RunMode` each time the buttons are pressed (further down in the Gamepad2 handler) it would be better to `setTargetPosition(0)` immediately after resetting the encoder then setting the `RunMode` to `RUN_TO_POSITION`. This way the mode is not being set more than necessary.